tests/integration_tests/resources/dsl/resumable_mgmtworker.yaml

```diff
@@ -30,3 +30,6 @@ node_templates:
     relationships:
       - type: cloudify.relationships.depends_on
         target: node1
+
+  node3:
+    type: cloudify.nodes.Root
```

Comment:
why this?

Reply:
to have at least one node `started` for workflow to be available and not mess with the tests

Reply:
but there already are node templates in this bp, we can just use those, right? I suppose they should be just installed (`self.deploy_application`), that's the primary interface of making nodes into the 'started' state